### ISSUE TITLE
Effects, Shaders & Other Graphics Settings

### ISSUE THREAD

**Issue by TRR95** [EKA2L1/EKA2L1#376]
Equal parts Critical and just something that would be nice to have. some games will benefit greatly from adding new graphics settings such as Post-Processing Effects and Custom Filtering Shaders. It would make it easier to play games for long periods of time on larger screens, while others will just look nicer overall.

I've tag this as one of the "Long Term Goals" of EKA2L1 since there is still work needs to be done on the graphics side of the emulator before we can add any additional effects or modifications to the games. this post will also keep track of the settings added to EKA2L1, both available right now and those that will be added when the graphics side is complete enough that it makes sense to add them

NOTE: Custom Filtering Shaders was originally requested by dmolina007#9747

- [ ] Stretch To Full Screen
- [ ] Filtering Shaders ( TXT, GLSL and or FX files)
- [ ] Vsync
- [x] Screen Buffer Sync
- [ ] Sharpen
- [ ] Deinterlacing
- [ ]  Texture Filtering
    - [x] Nearest Neighbor Filtering

**Comment by pent0** [comment]
From xlen:

> While playing around with games I've noticed that the low resolution is an issue with the gaming experience and well nearest neighbor filtering isn't really enough, maybe something more complex such as FSR could clean up the image.
This might be helpful:
https://gist.github.com/atyuwen/78d6e810e6d0f7fd4aa6207d416f2eeb#gistcomment-3873212

**Comment by TRR95** [comment]
From [kenobizero](https://github.com/kenobizero)
> MSAA better than FXAA, the graphic and text not blurry rather than FXAA at least its better with low resolution game like s60v2 and s60v3
https://github.com/EKA2L1/EKA2L1/issues/437
